retorch-llm-rp/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove if you are not using loggers

Reply:
I am using loggers in the experimentation classs